### DISCUSSION TITLE
Sign out of Windows 11?

### DISCUSSION THREAD

**Discussion by mschroner** [valinet/ExplorerPatcher#2790] [Q&A]
Hi, I'm not able to sign out of Windows 11 when I'm using ExplorerPatcher.  (When I right click on my icon, I don't get a menu.)  Is that expected?

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Comment by AMN-D** [comment]
you can actually just ctrl+alt+del and then ... you know u can sign out !

**Comment by mschroner** [comment]
@AMN-D Thanks, that's a great idea.  I'd still like to work with the team to figure out if this is a bug to fix, but that's a great workaround.

@pyrates999 Info:
Windows version: 22H2
Build: 22621.3007
ExplorerPatcher version: 22621.2861.62.2

Current settings:
![Advanced](https://github.com/valinet/ExplorerPatcher/assets/159200841/92b7be00-3fb0-4131-a3d8-c71dfe325d5c)
![Updates](https://github.com/valinet/ExplorerPatcher/assets/159200841/80675781-f377-42aa-8037-fd9606566c22)
![Other](https://github.com/valinet/ExplorerPatcher/assets/159200841/c122cd88-2d9d-429a-bf9e-5bd0e0dbf8b6)
![Weather](https://github.com/valinet/ExplorerPatcher/assets/159200841/1ba8e6bf-4e63-436c-ae2d-0a984a6b6a97)
![Window Switcher](https://github.com/valinet/ExplorerPatcher/assets/159200841/d2b5d31d-557f-410e-a556-e21e2f099ebf)
![Start Menu](https://github.com/valinet/ExplorerPatcher/assets/159200841/0ba3e2a2-4020-465e-b740-0b998eeca1b1)
![File Explorer](https://github.com/valinet/ExplorerPatcher/assets/159200841/feebc252-b429-427d-bfc9-0472d88d0403)
![System Tray](https://github.com/valinet/ExplorerPatcher/assets/159200841/6a788e72-d824-4b5d-9993-d248695de40e)
![Taskbar](https://github.com/valinet/ExplorerPatcher/assets/159200841/31ec06c8-788d-4612-8388-bd98a466fe05)

**Comment by pyrates999** [comment]
Check what happens when using the console in explorer patcher.  Open up the properties application for explorer patcher, go to advanced, and click on enable console.

Then try to log out using the start menu.  Please paste the results here.

**Reply by mschroner** [reply]
I enabled the console and a console window opened with a flashing cursor.  If I understood you correctly, then I just ignored that and tried to log out using the Start Menu.  I get to the point where I see my name, but when I click on it, it does not show the expected options.  Clicking on Power just below my name does show the expected options.  
Thanks for your help.

**Reply by mschroner** [reply]
Oh, when I clicked on the console, it seemed to activate it, and some text was generated.  I tried again to log out via Start Menu, but didn't get any different results, but here's the text.

```
Running on Windows 11, OS Build 10.0.22621.3007.
Loaded symbols
Setup user32 functions done
CAppResolverCacheBuilder::_AddUserPinnedShortcutToStart() = 47780
Patched taskbar transparency in newer OS builds
ITrayUIHost = 3B2318
explorer.exe!TrayUI_CreateInstance() = 19B6DC
Setup explorer functions done
Setup uxtheme functions done
[AC] rcMonitorAssignment = 247F1
[AC] rcWorkAssignment = 248A1
[AC] blockBegin = 24806
[AC] blockEnd = 248B7
[AC] Patched!
[CC] rcMonitorAssignment = 66EFB
[CC] rcWorkAssignment = 66FC8
[CC] blockBegin = 66F13
[CC] blockEnd = 66FE1
[CC] Patched!
[TC] rcMonitorAssignment = 5B6A0
[TC] rcWorkAssignment = 5B756
[TC] blockBegin = 5B6BA
[TC] blockEnd = 5B76F
[TC] Patched!
[TV] firstCallPrep = 2F665
[TV] firstCallCall = 2F66F
[TV] Patched!
[HC] match1 = 1D56D
[HC] match2 = 1D634
[HC] writeAt = 1D22E
[HC] cleanup = 1D645-1D65A
[HC] Patched!
PenMenuSystemTrayManager::GetDynamicSystemTrayHeightForMonitor() = 45D430
[SMA] match1 = 924F1
[SMA] match2 = 92530
[SMA] match3 = 5E73C
[SMA] CStartExperienceManager::GetMonitorInformation() = 1048B8
[SMA] CExperienceManagerAnimationHelper::Begin() = 71A66C
[SMA] CExperienceManagerAnimationHelper::End() = 2A41DC
[SMA] match7a in CStartExperienceManager::Hide() = 49185
[SMA] match7b in CStartExperienceManager::Hide() = 491D7
[SMA] CStartExperienceManager::OnViewCloaking() = 7B840
[SMA] match9 in CStartExperienceManager::OnViewHidden() = 5E73C
[SMA] CStartExperienceManager::OnViewUncloaked() = 7CD00
Setup twinui.pcshell functions done
Setup combase functions done
Setup twinui functions done
Setup stobject functions done
Setup bthprops functions done
Setup pnidui functions done
Setup shell32 functions done
Setup windows.storage functions done
[IME] Context menu patch status: 1
Setup inputswitch functions done
wil::details::FeatureImpl<__WilFeatureTraits_Feature_Servicing_TaskbarMultiMon_38545217>::__private_IsEnabled() = 10BEB8
Setup windowsudk.shellcommon functions done
Setup peopleband functions done
Installed hooks.
Play startup sound thread...
Signal shell ready...
Started "Signal shell ready" thread.
Open Start on monitor thread
Started "Open Start on current monitor" thread.
EP Service Window thread
Initialized taskbar centering module.
[Positioning] Added settings for monitor 0000000000020001 : 0
[Positioning] Added settings for monitor 0000000000010041 : 0
Registered Win+A, Win+B, and Win+N
Progman: 131624
Progman hook: 2756023
>>> Signal shell ready.
Setup sndvolsso functions done
Ended "Signal shell ready" thread.
0 0
Started "Play startup sound" thread.
Ended "Play startup sound" thread.
[Updates] Starting daemon.
[Updates] Configured update policy on this system: "Install updates automatically".
[Updates] Path to module: C:\Windows\dxgi.dll
[Updates] Checking against hash "22622.2861.62.2.c2b7c0292fff8608"
[Updates] Update URL: https://github.com/valinet/ExplorerPatcher/releases/latest/download/ep_setup.exe
[Updates] Hash of remote file is "22622.2861.62.2.c2b7c0292fff8608" (valid).
[Updates] Local version obtained from hash is 22622.2861.62.2.
[Updates] No updates are available.
[EnsureXAML] 31 ms.
```

**Reply by pyrates999** [reply]
Can you show a screen shot of what you get when you try to open the start menu to log out?

**Reply by mschroner** [reply]
Here are two screenshots, one of what happens when I mouseover or click on my user icon, and one when I mouseover the power icon (just to show the difference).
![Log out](https://github.com/valinet/ExplorerPatcher/assets/159200841/f1641a29-ea26-4ad5-ac64-bf23feca1e8f)
![Power](https://github.com/valinet/ExplorerPatcher/assets/159200841/504d4445-2c18-4db7-a806-64da70e92171)

**Reply by pyrates999** [reply]
Try moving the taskbar back to the bottom of the screen.  You can do so by right clicking on it and unticking **lock the taskbar**.

**Reply by mschroner** [reply]
I get the exact same result as the above images.

**Reply by pyrates999** [reply]
Did you run vivetool to enable or disable any windows features?

**Comment by milestprower92** [comment]
> Hi, I'm not able to sign out of Windows 11 when I'm using ExplorerPatcher. (When I right click on my icon, I don't get a menu.) Is that expected?

Microsoft recently removed the user menu from the windows 10 start menu

**Reply by AMN-D** [reply]
what do you mean i still have the user menu ? 
![image](https://github.com/valinet/ExplorerPatcher/assets/129135638/a1bf3dec-a612-49cd-82f6-846eb502e7d2)

**Reply by pyrates999** [reply]
Please list the build of windows 11 that the right click menu on the windows 10 start menu was removed on.

**Reply by milestprower92** [reply]
![imagen](https://github.com/valinet/ExplorerPatcher/assets/131640765/46194bee-c3a1-4c74-a247-b780dba53e95)
this is the menu that i was talking about

**Reply by pyrates999** [reply]
Ok, thanks.  Please wait for a developer to respond.
